include/ignition/gazebo/EntityComponentManager.hh

```diff
@@ -768,8 +768,10 @@ namespace ignition
           Entity _entity,
           const std::unordered_set<ComponentTypeId> &_types = {}) const;
 
+      /// This was originally a unique_ptr, but pybind11 was failing because it
+      /// was not able to copy the class
```

Comment:
I think this is a symptom of a bigger issue. Nowhere on the example we should be trying to copy the ECM. I think we need to understand where this is happening.

I tried changing it back to `unique_ptr` and confirmed that the example fails with:

```
Traceback (most recent call last):                          
  File "src/ign-gazebo/examples/python/helperFixture.py", line 64, in <module>
    helper.finalize()
RuntimeError: return_value_policy = copy, but the object is non-copyable!
```

I traced it back to this line:

https://github.com/ignitionrobotics/ign-gazebo/blob/ae36d833f01e7078ff8bb397bee5981a0e93ed13/src/TestFixture.cc#L165

If I comment it out the example runs (but it's incorrect as expected). I'm still trying to make sense of it.

Reply:
I see you reverted the pointer change, great! Let's also remove the comment.